docs/pages/dms/client.mdx

```diff
@@ -4,95 +4,147 @@ description: Learn how to create and configure an XMTP client
 
 # Create and configure an XMTP client
 
-The XMTP message API revolves around a network client that allows retrieving and sending messages to other network participants. A client must be connected to a wallet on startup. The client will request a wallet signature in two cases:
-
-1. To sign the newly generated key bundle. This happens only the very first time when key bundle is not found in storage.
-2. To sign a random salt used to encrypt the key bundle in storage. This happens every time the client is started (including the very first time).
+The XMTP message API revolves around a network client that allows retrieving and sending messages to other network participants. 
 
 ## Create a client
```

Comment:
Updated this section to combine https://docs.xmtp.org/dms/client#create-a-client with https://docs.xmtp.org/groups/build-group-chat#create-a-client-that-supports-group-chat to provide a common "Create a client" section that works for both 1:1 and group chat.

Please review the code samples in particular - I asked Cursor for help with updating the samples.